doc/19_Development_Tools_and_Details/08_Generic_Execution_Engine/README.md

```diff
@@ -1,7 +1,8 @@
 # Generic Execution Engine
 :::caution
 
-To use this feature, please enable the `PimcoreGenericExecutionEngineBundle` in your `bundles.php` file and install it accordingly with the following command:
+To use this feature, please update your `composer.json` adding `phpdocumentor/reflection-docblock` and `symfony/property-info` like in the suggest section. Enable the `PimcoreGenericExecutionEngineBundle` in your `bundles.php` file and install it accordingly with the following command:
```

Comment:
```suggestion
To use this feature, please update your `composer.json` by adding `phpdocumentor/reflection-docblock` and `symfony/property-info` like in the `suggest` section. Enable the `PimcoreGenericExecutionEngineBundle` in your `bundles.php` file and install it accordingly with the following command:
```
or maybe `by requiring` instead of `adding` to sound more "technical" and explain that it would be mandatory?